src/picongpu/include/particles/creation/creation.kernel

```diff
@@ -0,0 +1,307 @@
+/**
+ * Copyright 2015-2016 Heiko Burau, Macro Garten
```

Comment:
Yes please add the original authors if it is a copy past of an other file. Add your name as new name at the end of the list.